.travis.yml

```diff
@@ -43,7 +47,32 @@ before_install:
 - bundle install
 - bundle exec rake db:migrate
 - bundle exec rake lapis:api_keys:create_default
-- rails s -p 3000 >/dev/null &
+- rails s -p 3000 & 
+- tail -f log/test.log &
```

Comment:
Please revert to the previous thing: `rails s -p 3000 >/dev/null &`. Those two lines were only to make debugging easier.